sabakan/generator.go

```diff
@@ -633,6 +653,40 @@ func (g *Generator) getUnusedMachines(usedNodes []*cke.Node) []*Machine {
 	return unused
 }
 
+func (g *Generator) isTaintedInCluster(m *Machine) bool {
+	n, ok := g.k8sNodeMap[m.Spec.IPv4[0]]
+	if !ok {
+		// We can return false even if the Kubernetes Node's taints are unknown.
+		// This is because the conditions on the taints are not mandatory.
+		return false
+	}
+
+OUTER:
+	for _, t := range n.Spec.Taints {
+		if strings.HasPrefix(t.Key, "node.kubernetes.io/") || t.Key == "cke.cybozu.com/state" || t.Key == op.CKETaintMaster {
```

Comment:
I prefer to define `cke.cybozu.com/state` as const.

Reply:
I also prefer to use constants:-)
`sabakan/generator.go` contains so many string literals that I've given up defining them as constants in this PR.